app/src/main/java/com/eveningoutpost/dexdrip/stats/StatsResult.java

```diff
@@ -75,14 +77,22 @@ public StatsResult(SharedPreferences settings, long from, long to){
             avg = 0;
         }
 
+        if(getTotalReadings() > 0){
+            cursor= db.rawQuery("select ((count(*)*(sum(calculated_value * calculated_value)) - (sum(calculated_value)*sum(calculated_value)) )/((count(*)-1)*(count(*))) ) from bgreadings  where timestamp >= " + from + " AND timestamp <= " + to + " AND calculated_value > " + DBSearchUtil.CUTOFF + " AND snyced == 0", null);
+            cursor.moveToFirst();
+            stdev = cursor.getDouble(0);
+            stdev = Math.sqrt(stdev);
```

Comment:
Sorry @kskandispersonal, I'm on a train and the notification about Jon's response arrived late.
Either lazy-loading or just calculating it when the setting is activated sounds good.
Nice doing this in SQL btw. (as SQLite does not offer this) :)